### PR TITLE
Remove unnecessary kotlin path declaration in gradle build file

### PR DESCRIPTION
**Pull request by glefloch**
As mentionned by @aloubyansky, it's not necessary to declare source path and output path for kotlin. 

I removed it from tests and from codestart template.

### CODE REVIEW

**Review by aloubyansky — APPROVED** [review]

File: integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle [hunk 1]
```diff
@@ -23,33 +23,16 @@ dependencies {
...
-}
-
 allOpen {
     annotation("javax.ws.rs.Path")
     annotation("javax.enterprise.context.ApplicationScoped")
     annotation("io.quarkus.test.junit.QuarkusTest")
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_11
```
Comment (gastaldi) [inline integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle:41]: Why was this removed?

Comment (glefloch) [reply integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle:41]: We used to not specify java version in sample project as Gradle test were ran against jdk 8. But this is not the case anymore. I don't know what if we should keep it or not..

Comment (ia3andy) [reply integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle:41]: @glefloch 👆

Comment (glefloch) [reply integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle:41]: @ia3andy should I revert this change?

Comment (ia3andy) [reply integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle:41]: @glefloch  I don't know 😅
